instrumentation/opentelemetry-instrumentation-pymemcache/tests/test_pymemcache.py

```diff
@@ -33,6 +35,14 @@
 TEST_HOST = "localhost"
 TEST_PORT = 117711
 
+pymemcache_version = get_package_version.parse(pymemcache_package_version)
+pymemcache_version_lt_2 = pymemcache_version < get_package_version.parse(
```

Comment:
Maybe include a summary of telemetry-related differences between the versions in comments? This one would specifically be about batching of the `set_many` call.

Reply:
Will do